850.split-ambiguities/m.yaml

```diff
@@ -313,6 +313,9 @@
 - { name: monocle, wwwpart: packetstormsecurity, setname: monocle-host-discovery }
 - { name: monocle, addflag: unclassified }
 
+- { name: morpheus, wwwpart: [morpheus.gitlab.io, morpheus.lab], setname: morpheus-modeling }
```

Comment:
There are like 4 projects, a rule is needed for each. You also need to merge `morpheus-modeling` to `morpheus` so it also passes through this very split rule. This way the project is split reliably even if target names are changed.

Reply:
Updated but wasn't sure on best name for https://github.com/cwenger/Morpheus. May need to be improved.